modules/adding-node-iso-configs.adoc

```diff
@@ -0,0 +1,162 @@
+|An array of host configuration objects.
+
+|hosts:
+  interfaces:
+    name:
+|The name of an interface on the host.
+|String.
+
+|hosts:
+  interfaces:
+    macAddress:
+|The MAC address of an interface on the host.
+|A MAC address such as the following example: `00-B0-D0-63-C2-26`.
+
+|hosts:
+  rootDeviceHints:
+|Enables provisioning of the {op-system-first} image to a particular device.
+The node-adding tool examines the devices in the order it discovers them, and compares the discovered values with the hint values.
+It uses the first discovered device that matches the hint value.
+|A dictionary of key-value pairs.
+For more information, see "Root device hints" in the "Setting up the environment for an OpenShift installation" page.
+
+|hosts:
+  rootDeviceHints:
+    deviceName:
+|The name of the device the {op-system} image is provisioned to.
+|String.
+
+|hosts:
+  networkConfig:
+|The host network definition.
+The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
+|A dictionary of host network configuration objects.
+
+|====
+
+
+[id="adding-node-iso-flags-config_{context}"]
+== Command flag options
+
+You can use command flags with the `oc adm node-image create` command to configure the nodes you are creating.
+
+The following table describes command flags that can be used only when creating a single node:
+
+.Single-node only command flags
+[cols=".^4,.^3,.^2",options="header"]
+|====
+|Flag|Description|Values
+
+|`-c`, `--cpu-architecture`
+|The CPU architecture to be used to install the node.
+This flag can be used to create only a single node, and the `--mac-address` flag must be defined.
+|String
+
+|`--hostname`
+|The hostname to be set for the node.
+This flag can be used to create only a single node, and the `--mac-address` flag must be defined.
+|String
+
+
+|`-m`, `--mac-address`
+|The MAC address used to identify the host to apply configurations to.
+This flag can be used to create only a single node, and the `--mac-address` flag must be defined.
+|String
+
+|`--network-config-path`
+|The path to a YAML file containing NMState configurations to be applied to the node.
+This flag can be used to create only a single node, and the `--mac-address` flag must be defined.
+|String
+
+|`--root-device-hint`
+|A hint for specifying the storage location for the image root filesystem. The accepted format is `<hint_name>:<value>`.
+This flag can be used to create only a single node, and the `--mac-address` flag must be defined.
+|String
+
+|`-k`, `--ssh-key-path`
+|The path to the SSH key used to access the node.
+This flag can be used to create only a single node, and the `--mac-address` flag must be defined.
+|String
+|====
+
+The following table describes command flags that are not limited to the single-node use case:
+
+.Additional command flags
```

Comment:
The two sections for the flags are really nice, I'd just swap the order, as it seems more natural to me to show to the reader the common flags first

Reply:
"general command flags" sounds reasonable to me